com.unity.ml-agents/Runtime/Communicator/GrpcExtensions.cs

```diff
@@ -59,6 +60,24 @@ public static AgentInfoProto ToAgentInfoProto(this AgentInfo ai)
             return agentInfoProto;
         }
 
+        public static List<ObservationSummary> GetObservationSummaries(this AgentInfoActionPairProto infoActionPair)
```

Comment:
The method name is very explicit but maybe a method comment?

Reply:
Updated.